catroid/src/main/java/org/catrobat/catroid/io/ProjectAndSceneScreenshotLoader.java

```diff
@@ -115,6 +120,41 @@ public File getScreenshotFile(String projectName, String sceneName, boolean isBa
 		return screenshotLoader.getScreenshotFile();
 	}
 
+	public String getScreenshotScene(File projectDir) {
+		FilenameFilter filter = new FilenameFilter() {
+
+			public boolean accept(File f, String name) {
+				return name.endsWith("screenshot.png");
+			}
+		};
+		File[] projectScreenshots = projectDir.listFiles(filter);
+		if (projectScreenshots == null || projectScreenshots.length != 0) {
+			return null;
+		}
+		List<File> screenshots = new ArrayList<>();
+		for (File scene : projectDir.listFiles()) {
+			File[] sceneScreenshots = scene.listFiles(filter);
+			if (sceneScreenshots != null && sceneScreenshots.length > 0) {
+				screenshots.addAll(Arrays.asList(scene.listFiles(filter)));
+			}
+		}
+		if (screenshots.size() < 1) {
+			return null;
+		}
+		Collections.sort(screenshots, new Comparator<File>() {
+			@Override
+			public int compare(File screenshot2, File screenshot1) {
+				return Long.compare(screenshot1.lastModified(), screenshot2.lastModified());
+			}
+		});
+		for (File screenshot : screenshots) {
+			if (screenshot.getName().equals("manual_screenshot.png")) {
```

Comment:
again, please no hard coded values, `manual_screenshot.png` is already defined in `StageListener.java` line 106,
could you move it to Constants.java and refactor its usage?